andela/healthchecks_akira
# #139551331 User requires a new dashboard tab for unresolved stuff

#### What does this PR do?
 * When a User's job goes down, the user is sent a notification email informing him of the same.
 * Apart from the email notifications, the user should will also have a separate dashboard view of  their failed jobs.
#### Description of Task to be completed?
 * User should have a separate tab for viewing the failed jobs apart from the checks tab that allows the user to view all the jobs.
#### How should this be manually tested?
  * run the following code `./manage.py runserver` to run the development server.
  * Login as a user and you should be able to see a tab for the failed jobs as shown below in the screenshots section.
#### What are the relevant pivotal tracker stories?
 * #139551331 
#### Screenshots
   <img width="1192" alt="failed" src="https://cloud.githubusercontent.com/assets/18587173/23405509/9c471cb2-fdcb-11e6-97f6-2fab846e43e9.png">
